client/src/services/discipline.ts

```diff
@@ -0,0 +1,48 @@
+import axios, { AxiosInstance } from 'axios';
+import { message } from 'antd';
+import { IDiscipline } from '../modules/Discipline/model';
+import { getServerAxiosProps } from '../utils/axios';
+
+export class DisciplineService {
+  private axios: AxiosInstance;
+
+  constructor(token?: string) {
+    this.axios = axios.create(getServerAxiosProps(token));
+  }
+
+  async getAllDisciplines() {
+    try {
+      const result = await this.axios.get('/api/v2/disciplines');
```

Comment:
please use `DisciplineApi` from `api`. We generate client from swagger and don't need to construct urls/services anymore.